packages/fast-check/src/arbitrary/_internals/mappers/KeyValuePairsToObject.ts

```diff
@@ -51,11 +48,14 @@ export function keyValuePairsToObjectUnmapper<T>(value: unknown): ObjectDefiniti
   if (!hasNullPrototype && !hasObjectPrototype) {
     throw new Error('Incompatible instance received: should be of exact type Object');
   }
-  if (safeObjectGetOwnPropertySymbols(value).length > 0) {
-    throw new Error('Incompatible instance received: should contain symbols');
-  }
-  if (!safeEvery(safeObjectGetOwnPropertyNames(value), buildIsValidPropertyNameFilter(value))) {
+  const propertyDescriptors = safeReflectOwnKeys(value).map((key): [PropertyKey, PropertyDescriptor] => [
```

Comment:
Prefer using my safe version of map: safeMap. It resists any poisoning, given fast-check can easily makes a piece of code crazy and destroy every proto, we have these copied functions to back us from any risks